src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -22,7 +22,7 @@
 /**
  * Parses input arguments and creates a new AddCommand object
  */
-public class AddCommandParser implements Parser<AddCommand> {
+public class AddCommandParser implements CommandParser<AddCommand> {
```

Comment:
Wait why did you do this haha I didn't change this part for addcommandparser and I feel like if its changed to commandparser the add function might not work properly :(